virttest/vt_iothread.py

```diff
@@ -103,7 +103,7 @@ def request_iothread(self, iothread):
             raise ValueError("Not support request specific iothread")
 
 
-class MultipeerRoundRobinManager(IOThreadManagerBase):
+class MultiPeerRoundRobinManager(IOThreadManagerBase):
```

Comment:
Maybe it should be covered in the previous commit `vt_iothread: New class added`, not here.